### PR TITLE
Use ellipsis instead of *** wilcards and provide -ELLIPSIS option.

### LINKED ISSUE #69: Python doctest use "..." and we use "***" to match arbitrary text
Thoughts on changing to `...` before we get too popular to change? ;-)

`...` looks nice for shortening a long vector output.  Maybe `***` is more natural as "wildcard".

OTOH, from [reading this](https://docs.python.org/3/library/doctest.html#doctest-directives), it looks like you have to enable with:
````
#doctest: +ELLIPSIS
````

### PR DESCRIPTION
**Pull request by catch22**
Fix #69.

### CODE REVIEW

File: inst/doctest.m [hunk 1]
```diff
@@ -146,15 +146,18 @@
 % before comparison.  A stricter mode where "internal whitespace" must
 % match is available:
 %
-% >> fprintf('a   b\nc   d\n')    % doctest: -NORMALIZEDWHITESPACE
+% >> fprintf('a   b\nc   d\n')    % doctest: -NORMALIZEWHITESPACE
```
Comment (cbm755) [inline inst/doctest.m]: ah, sorry.  thanks for this

**Comment by cbm755** [comment]
This looks great!  Thanks for the refactor.  +1

**Comment by cbm755** [comment]
BTW, in case its helpful: you can put "Fixed #69" in the longer part of the commit message to auto-close issues.

**Comment by catch22** [comment]
Thanks!

**Pull request merged**